### ISSUE TITLE
[RPM] Wrong permissions for /var/ossec/queue/rids/* after removing

### ISSUE THREAD

**Issue by jesusgn90** [wazuh/wazuh-packages#76]
Hello team, we have a problem with the files in `/var/ossec/queue/rids/*` after `yum remove wazuh-agent`.

#### Steps to reproduce:

1. `yum install wazuh-agent`
2. Register your agent to an active Wazuh manager.
3. `yum remove wazuh-agent`
4. Now you can install an agent (`yum install wazuh-agent`) or a manager (`yum install wazuh-manager`). Each way ends in a different error.

#### Impact:

At this point we have two different situations, both come from wrong permissions for `/var/ossec/queue/rids/*`:

1. If we are going to install a Wazuh agent, it adds a new counter after the registration process, this means each new installation will add a new counter:

```
[root@worker rids]# ll -h
total 8,0K
-rw-r--r-- 1 ossec ossec 7 oct  9 16:10 017
-rw-r--r-- 1 ossec ossec 0 oct  9 16:12 018
-rw-r--r-- 1 ossec ossec 0 oct  9 16:12 018
-rw-r--r-- 1 ossec ossec 0 oct  9 16:12 020
-rw-r--r-- 1 ossec ossec 0 oct  9 16:12 021
-rw-r--r-- 1 ossec ossec 0 oct  9 16:12 022
...
-rw-r--r-- 1 ossec ossec 7 oct  9 16:20 sender_counter
```

2. If we are going to install a Wazuh manager, it has wrong permissions and some errors regarding to this:

```
[root@worker rids]# cat /var/ossec/logs/ossec.log | grep -i -E "(ERROR|WARNING|CRITICAL)"
2018/10/09 16:25:20 ossec-remoted: ERROR: Unable to open agent file. errno: 13
2018/10/09 16:25:20 ossec-remoted: CRITICAL: (1103): Could not open file '/queue/rids/sender_counter' due to [(13)-(Permission denied)].
```

Also `remoted` is down...

```
ps aux | grep ossec
ossec    17742  0.2  0.1 561824  4740 ?        Sl   16:25   0:00 /var/ossec/bin/wazuh-db
root     17755  0.0  0.0  29612  2672 ?        Sl   16:25   0:00 /var/ossec/bin/ossec-execd
ossec    17762  0.3  0.3 678608 12612 ?        Sl   16:25   0:00 /var/ossec/bin/ossec-analysisd
root     17766  5.5  0.1 112548  5284 ?        Sl   16:25   0:05 /var/ossec/bin/ossec-syscheckd
root     17776  0.0  0.0 398264  2792 ?        Sl   16:25   0:00 /var/ossec/bin/ossec-logcollector
ossec    17788  0.0  0.0  29580  2556 ?        Sl   16:25   0:00 /var/ossec/bin/ossec-monitord
root     17791  0.1  0.1 351368  7344 ?        Sl   16:25   0:00 /var/ossec/bin/wazuh-modulesd
```

From my view, this is a bug for both agent and manager:

- For the agent we have non-desired files, and we are accumulating files installation by installation. 
- For the Wazuh manager, we have a critical error and `remoted` down.

Best regards,
Jesús

**Comment by BraulioV** [comment]
Hi @jesusgn90,

this was solved in Wazuh v3.7.0.

Regards,
Braulio.

**Comment by crycookie** [comment]
Still  happening to me with 3.8.2 on Centos 7.4. 
Steps to reproduce:
1. install WZH agent
2. Remove agent
3. install manager.
